src/instance/discord/commands/punishments.ts

```diff
@@ -218,12 +218,12 @@ async function handleMuteAddInteraction(
 
     type: PunishmentType.MUTE,
     reason: reason,
-    till: punishTill
+    till: Date.now() + muteDuration
   }
 
   application.punishedUsers.punish(event)
   application.clusterHelper.sendCommandToAllMinecraft(
-    `/guild mute ${username} ${PunishedUsers.tillTimeToMinecraftDuration(punishTill)}`
+    `/guild mute ${username} ${PunishedUsers.durationToMinecraftDuration(muteDuration)}`
   )
   application.clusterHelper.sendCommandToAllMinecraft(
```

Comment:
`sendCommandToAllMinecraft` will broadcast message to be sent by ALL connected bridge accounts. This could result in multiple private messages being sent to the target if multiple bridges are connected together.

Reply:
Another issue with privately messaging targets, is that they may have DMs disabled.